doc/source/api/variants.rst

```diff
@@ -46,19 +46,28 @@ There remain some outstanding challenges with this model of small variants. For
 Variants Schema Entities
 ------------------
 
-While the Variant schema is based on VCF, it allows for more versatile interaction with the data. Instead of sending whole VCF files, the server can send information on specific variants or genomic regions instead. And instead of getting the whole genotype matrix, it's possible to just get details for a single individual.
+While the Variant schema is based on VCF, it allows for more versatile interaction with the data. Instead of sending whole VCF files, the server can send information on specific variants or genomic regions instead. And instead of getting the whole genotype matrix, it's possible to just get details for one or more specified individuals.
 
-The API uses four main entities to represent variants. The following diagram illustrates how these entities constitute the genotype matrix. 
+The API uses four main entities to represent variants. The following diagram illustrates how these entities relate to each other to constitute the genotype matrix. 
 
 .. image:: /_static/variant_objects.svg
 
-The atomic :avro:record:`Call` encodes the genotype of an individual with respect to a variant, as determined by some analysis of experimental data. The other objects can be thought of as collections of Calls that have something in common:
+The lowest-level entity is a Call:
+
+    * a :avro:record:`Call` encodes the genotype of an individual with respect to a variant, as determined by some analysis of experimental data.
+
+The other entities can be thought of as collections of Calls that have something in common:
 
     * a :avro:record:`VariantSet` supports working with a collection of Calls intended to be analyzed together.
-    * a :avro:record:`Variant` supports working with the subset of Calls in a VariantSet for the same site of two or more alleles.
-    * a :avro:record:`CallSet` supports working with the subset of Calls in a VariantSet that were generated by the same analysis of the same sample.
+    * a :avro:record:`Variant` supports working with the subset of Calls in a VariantSet that are at the same site and are described using the same set of alleles. The Variant entity contains:
+    
+        * a variant description: a potential difference between experimental DNA and a reference sequence, including the site (position of the difference) and alleles (how the bases differ)
```

Comment:
I tend to think of the alleles as logically part of the site (i.e. the specification of a site includes the observed alleles), however, these terms are largely passed down by oral history, so I'm not adamant.